frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AppleBusinessManagerPage/components/RenewAbmModal/RenewAbmModal.tsx

```diff
@@ -106,14 +102,7 @@ const RenewAbmModal = ({
                 graphicName="file-p7m"
                 message="ABM token (.p7m)"
                 onFileUpload={onSelectFile}
-                filePreview={
-                  tokenFile && (
-                    <FileDetails
-                      details={{ name: tokenFile.name }}
-                      graphicName="file-p7m"
-                    />
-                  )
-                }
+                fileDetails={tokenFile ? { name: tokenFile.name } : undefined}
```

Comment:
Looks like the graphic name may have been accidentally removed here?

Reply:
Passed in from line 102